test/dotcom_web/live/trip_planner_test.exs

```diff
@@ -4,401 +4,249 @@ defmodule DotcomWeb.Live.TripPlannerTest do
   import Mox
```

Comment:
📓 Rather than looking at the diff, I would review the file in its entirety since I completely rewrote it.